src/Helper/EntriesExport.php

```diff
@@ -432,6 +433,31 @@ private function produceTxt()
         );
     }
 
+    /**
+     * Dump a Markdown file.
+     *
+     * @return Response
+     */
+    private function produceMd()
```

Comment:
Can't we apply typing for new method?
```suggestion
     */
    private function produceMd(): Response
```